payroll/views.py

```diff
@@ -118,12 +156,15 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         department_statistics = self.aggregate_department_statistics()
 
+        payroll_chart_data = super()._make_pie_chart("payroll-expenditure-chart")
```

Comment:
Since this is the same between units and departments, you call this function and add the output to the context on the parent `EmployerView` class.